app/models/plan_cost_decorator.rb

```diff
@@ -97,12 +97,17 @@ def reference_premium_for(member)
   end
 
   def rate_lookup(the_plan, start_on_date, age, member, benefit_group)
-    rate_value = if @multiple_rating_areas
+    rate_value = if @multiple_rating_areas && the_plan.health?
```

Comment:
again, per saadi's image - dental plans use rating areas. This would use the rate lookup that ignores rating area.

Am I incorrect?